icon-loader.js

```diff
@@ -9,6 +9,8 @@ module.exports = function shopifyIconLoader(source) {
     return fill.includes('#FFF') ? 'fill="currentColor"' : '';
   });
 
+  // Issue with ESLint recognizing this as needing an object destructure
+  // eslint-disable-next-line prefer-destructuring
```

Comment:
It seems like the [`enforceForRenamedProperties`](https://github.com/eslint/eslint/blob/master/lib/rules/prefer-destructuring.js#L117-L119) option doesn't take into consideration the state of the array option.